test-tests/shared/src/test/scala/zio/test/SpecSpec.scala

```diff
@@ -187,5 +173,5 @@ object SpecSpec extends ZIOBaseSpec {
         }
       )
     }
-  )
+  ) @@ TestAspect.ignore // TODO Rewrite these to use final Summary instead of ExecutedSpec
```

Comment:
Can we minimize this any more? There are a lot of tests in here that are making assertions about behavior that is not related to the test rendering.

Reply:
Applied to only specific failing cases now.